doc/manual/manual.tex

```diff
@@ -2286,6 +2286,58 @@ \subsection{Advection Stabilization}
 \end{itemize}
 
 
+Both add additional terms to the temperature (or compositional field equation). We will discuss
+the case for the temperature equation here. The compositional fields only differ in having
+a zero conductivity, fewer right-hand side terms, and $\rho C_p=1$. The strong form of the temperature equation reads
+\[
+ \rho C_p \frac{\partial T}{\partial t} + \rho C_p \mathbf{u} \cdot \nabla T - \nabla \cdot k\nabla T = F,
+\]
+where $F$ is the combination of source and reaction terms,
+while the weak form (with test function $\varphi$) becomes:
+\[
+a(T,\varphi) =
+ \left(\rho C_p \frac{\partial T}{\partial t}, \varphi \right)
+ + \left(\rho C_p \mathbf{u} \cdot \nabla T, \varphi \right)
+ + \left( k \nabla T, \nabla \varphi \right) = (F,\varphi) = f(\varphi).
+\]
+
+\subsubsection{SUPG Stabilization}
+
+For streamline upwind/Petrov-Galerkin (SUPG) (see for example \cite{JohnKnobloch2006,step-63}), we amend the weak form with a cell-wise
```

Comment:
What does that mean, you "amend" the weak form? This is just added on top of the original left- and right-hand side of the equation?